internal/provider/resource_topic_config.go

```diff
@@ -62,6 +62,16 @@ func (t topicConfigResourceType) GetSchema(ctx context.Context) (tfsdk.Schema, d
 					tfsdk.RequiresReplace(),
 				},
 			},
+			"key_schema_version": {
+				MarkdownDescription: "The keySchemaVersion this stream configuration is associated with",
```

Comment:
Maybe more clear messaging.
The schemaversion this topic config supports for the key.

The schemaversion this topic config supports for the value.

Reply:
I fixed it.